fr/development/routing.rst

```diff
@@ -830,7 +830,7 @@ utilisés lors de la connexion des routes. Par exemple, si votre action
 renommer vos actions::
 
     $routes->resources('Articles', [
-        'actions' => ['update' => 'put', 'create' => 'create']
+        'actions' => ['update' => 'put', 'add' => 'create']
```

Comment:
Well reading the line below this one, it should actually be `'actions' => ['update' => 'put', 'create' => 'add']`